test/token/ERC20/ERC20Base.behavior.js

```diff
@@ -51,11 +52,47 @@ const describeBehaviorOfERC20Base = function ({ deploy, supply }, skips) {
     });
 
     describe('#transfer', function () {
-      it('todo');
+      it('transfers amount from a to b', async function(){
+        const amount = ethers.constants.Two;
+        const zero = ethers.constants.Zero;
+        await mint(spender.address, amount);
+        expect(await instance.callStatic.balanceOf(receiver.address)).to.equal(zero);
+        expect(await instance.callStatic.balanceOf(spender.address)).to.equal(amount);
+
+        await instance.connect(spender).transfer(receiver.address, amount);
```

Comment:
Use the `changeTokenBalances` matcher here: https://ethereum-waffle.readthedocs.io/en/latest/matchers.html

No need to check that initial balance is 0.